src/views/credits/credits.jsx

```diff
@@ -26,6 +26,11 @@ var Credits = React.createClass({
                         <img src="//cdn.scratch.mit.edu/get_image/user/3581881_170x170.png" alt="Carl Avatar" />
                         <span className="name">Carl Bowman</span>
                     </li>
+                                        
+                     <li>
```

Comment:
Looks great! One last issue: It looks like your indentation is off by one space.

Reply:
Fixes 😄